README.md

```diff
@@ -1,3 +1,22 @@
 # EsaBusModule
 
 This is a repository for all things around the EsaBusModule, an Arduino Nano compatible development board for the ESA 1919 and ESA 5000 electric scooters.
+
+## How does it work?
+
+The added board sends different commands to the motor controller. You can set a new max. speed or enable lock mode (immobilizer) without changing the original dashboard or controller.
+It makes use of throttle & brake levers and the button on the dashboard.
+
+## How to use?
+
+There are 3 speed settings LOW (20 km/h), MEDIUM (25 km/h) and HIGH (30 km/h) and the option to enable/disable lock mode.
+If you use the original hardware, the max. speed after power on is always 20 km/h, so you have to set a new speed every time.
+A successful change of speed is shown by switching to eco mode for a short time (green LED lights up).
+You can not change speed limit of eco mode, it will stay at 15 km/h.
+
+- 3x button or 5x button = 20 km/h
```

Comment:
I'd prefer if there was still a way to reset the speed limit by pulling the brake for a few seconds or so. Would you mind adding that? :) The button press codes could still remain this way.

Reply:
I thought about that, too.
We should have an option to disable tuning very inconspicuous, but without doing it accidentally.
I'll add 5 seconds full brake or something like that.

Reply:
I just added an option to reset max speed by holding down throttle and brake for 3 seconds.
This should be fast and easy enough, no one becomes suspicious and it shouldn't happen by accident in normal driving.

I guess we could even remove the other options of detuning (3x / 5x button).

Reply:
I don't see that change here. Will it be in a separate pull request?

Reply:
Sorry, i didn't push it yet... 😅